<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="4" passed="3">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="7789" started-at="2018-12-14T07:12:20Z" finished-at="2018-12-14T07:12:28Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="7789" started-at="2018-12-14T07:12:20Z" finished-at="2018-12-14T07:12:28Z">
      <class name="com.qapitol.qa.test.DevicesTest">
        <test-method status="PASS" signature="getToken()[pri:0, instance:com.qapitol.qa.test.DevicesTest@1753acfe]" name="getToken" is-config="true" duration-ms="6508" started-at="2018-12-14T12:42:20Z" finished-at="2018-12-14T12:42:27Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getToken -->
        <test-method status="PASS" signature="header()[pri:0, instance:com.qapitol.qa.test.DevicesTest@1753acfe]" name="header" is-config="true" duration-ms="6" started-at="2018-12-14T12:42:27Z" finished-at="2018-12-14T12:42:27Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- header -->
        <test-method status="PASS" signature="createDevice()[pri:1, instance:com.qapitol.qa.test.DevicesTest@1753acfe]" name="createDevice" duration-ms="468" started-at="2018-12-14T12:42:27Z" finished-at="2018-12-14T12:42:27Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createDevice -->
        <test-method status="PASS" signature="header()[pri:0, instance:com.qapitol.qa.test.DevicesTest@1753acfe]" name="header" is-config="true" duration-ms="14" started-at="2018-12-14T12:42:27Z" finished-at="2018-12-14T12:42:27Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- header -->
        <test-method status="PASS" signature="getDevice()[pri:2, instance:com.qapitol.qa.test.DevicesTest@1753acfe]" name="getDevice" duration-ms="158" started-at="2018-12-14T12:42:27Z" finished-at="2018-12-14T12:42:27Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getDevice -->
        <test-method status="PASS" signature="header()[pri:0, instance:com.qapitol.qa.test.DevicesTest@1753acfe]" name="header" is-config="true" duration-ms="17" started-at="2018-12-14T12:42:27Z" finished-at="2018-12-14T12:42:27Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- header -->
        <test-method status="PASS" signature="updateDevice()[pri:3, instance:com.qapitol.qa.test.DevicesTest@1753acfe]" name="updateDevice" duration-ms="183" started-at="2018-12-14T12:42:27Z" finished-at="2018-12-14T12:42:27Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- updateDevice -->
        <test-method status="PASS" signature="header()[pri:0, instance:com.qapitol.qa.test.DevicesTest@1753acfe]" name="header" is-config="true" duration-ms="13" started-at="2018-12-14T12:42:27Z" finished-at="2018-12-14T12:42:27Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- header -->
        <test-method status="FAIL" signature="deleteDevice()[pri:4, instance:com.qapitol.qa.test.DevicesTest@1753acfe]" name="deleteDevice" duration-ms="394" started-at="2018-12-14T12:42:27Z" finished-at="2018-12-14T12:42:28Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[ status code is not 404 expected [404] but found [200]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError:  status code is not 404 expected [404] but found [200]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:389)
	at com.qapitol.qa.test.DevicesTest.deleteDevice(DevicesTest.java:115)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- deleteDevice -->
      </class> <!-- com.qapitol.qa.test.DevicesTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
